nodes/nodes.go

```diff
@@ -94,13 +104,28 @@ func (r *Report) Print() {
 
 // AddToReport adds the given node to the report
 func (r *Report) AddToReport(node Node) {
-	v, _ := node.Version()
+	var (
+		v, _      = node.Version()
+		badBlocks = node.BadBlocks()
+	)
+	// Add bad blocks to report
+	for _, block := range badBlocks {
+		r.BadBlocks = append(r.BadBlocks,
+			&badBlockJson{
+				Client: node.Name(),
+				Hash:   block.Hash,
+				RLP:    block.RLP,
```

Comment:
We might want to serialize the bad block RLP to some file, instead of serving it in the report every 10 seconds though. blocks are pretty large